a10_octavia/controller/worker/tasks/utils.py

```diff
@@ -99,3 +99,19 @@ def dash_to_underscore(my_dict):
         return item_dict
     else:
         return my_dict
+
+
+def ip_str_to_int(ip_str):
```

Comment:
It's better to use the socket library designed for this so that all edge cases are handled. Also less code we have to manage.

See: https://docs.python.org/3/library/socket.html#socket.inet_ntoa and https://docs.python.org/3/library/socket.html#socket.inet_aton

```socket.inet_ntoa(struct.pack("<L", '10.0.0.1'))``` For string -> int 

```struct.unpack("<L", socket.inet_aton(addr))``` Where addr is some hex, int, or octal

Reply:
will change to use this API